Neos.Flow/Classes/Security/AccountIdentifier.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Neos\Flow\Security;
+
+class AccountIdentifier {
```

Comment:
Hey @sorenmalling, 
Would be good to have the newly introduced String-value objects look similar. Please have a look at https://github.com/neos/neos-development-collection/blob/master/Neos.Media/Classes/Domain/ValueObject/Configuration/Label.php - I suggest to adjust this one and the others introduced: No getter, to string method, final ...